lib/web/apiserver_test.go

```diff
@@ -2566,6 +2566,81 @@ func TestCheckAccessToRegisteredResource(t *testing.T) {
 	}
 }
 
+func TestAuthExport(t *testing.T) {
```

Comment:
Let's test auth.ExportAuthorities in the auth package.

By all means keep this test, but here we should only be concerned that everything is wired correctly, whereas in `package auth` we want to test the export logic.

Reply:
Bump.

Reply:
Tests were added